src/config_brain_stats.js

```diff
@@ -116,7 +116,8 @@ brain.stats.init = function() {
       room: {}
     }
   };
-  let rolesNames = _(Game.creeps).map(c => c.memory.role).countBy(function(r) { return r; }).value();
+  let rolesNames = _.map(Game.creeps, 'memory.role', _.countBy(function(r) { return r; }));
```

Comment:
the actual version is a loadash chain so don't really need to change it :) and it's surely slower.